tox.ini

```diff
@@ -5,8 +5,8 @@ envlist = linter, unit
 [testenv:linter]
 basepython = python3
 commands =
-  {envpython} -m black --config black.ini --check .
-  {envpython} -m flake8 --select=E,W,F --ignore=E203,W503 --max-line-length=100 .
+  {envpython} -m black --config black.ini --check dbt tests
+  {envpython} -m flake8 --select=E,W,F --ignore=E203,W503 --max-line-length=100 dbt tests
```

Comment:
Clarifying for posterity: this change is necessary if you develop with a virtualenv in the same directory as the source code. Otherwise `black` and `flake8` will lint and format the site-packages of your Python installation which is both unnecessary and time-consuming. Andre and I learned this the hard way.